### PR TITLE
Remove non spatial

### PR DESCRIPTION
**Pull request by friedjoff**

### CODE REVIEW

File: src/components/LayerTree/LayerTree.js [hunk 1]
```diff
@@ -240,26 +221,29 @@ class LayerTree extends Component {
...
-        }`}
-        className={classNameToggle}
-        onClick={() => {
-          this.onInputClick(layer, layer.getChildren().length);
-        }}
+      <div
+        role="button"
+        tabIndex={0}
+        className="rs-layer-tree-toggle"
+        title={title}
```
Comment (edellucien) [inline src/components/LayerTree/LayerTree.js]: Could we add here 'aria-expanded' for accessibility:
https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/expanded/

Comment (edellucien) [reply src/components/LayerTree/LayerTree.js]:
```suggestion
        title={title}
        aria-expanded={expandedLayerNames.includes(layer)}
```

**Comment by friedjoff** [comment]
The broken Netlify build has been fixed, but the preview seems still broken. Anybody any idea?

**Comment by friedjoff** [comment]
Should work now. Problem was solved by switching all CSS class names to `rs-` prefix.

**Comment by friedjoff** [comment]
> The LayerTree does not work anymore: The checkboxes and radio button do not react.

Finally fixed by removing one line.

**Comment by friedjoff** [comment]
@oterral as discussed I've removed the TopicList component.

**Pull request merged**